cvmfs/sql_impl.h

```diff
@@ -218,10 +218,10 @@ bool Database<DerivedT>::FileReadAhead() {
     // Read-ahead is known to fail on tmpfs with EINVAL
     // EINVAL = "readahead() cannot be applied to that a file type"
     // Don't consider it a fatal error.
-    if (retval != 0 && errno != EINVAL) {
+    if (retval != 0 && errno != EINVAL && errno != ENOSYS) {
```

Comment:
Can you add to the comment above explaining the circumstances in which we might expect an ENOSYS?